### PR TITLE
feat(frontend): SubDag Node and mock samples. Partial #6419. Fix #6302. Fix #6398

### LINKED ISSUE #6302: Mock more get pipelineSpec response

### LINKED ISSUE #6398: Support breadcrumb to navigate among sub-DAGs.

### PR DESCRIPTION
**Pull request by zijianjoy**
**Description of your changes:**

 Partial #6419. Fix #6302. Fix #6398

subdagstyle

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: frontend/src/components/graph/SubDagLayer.tsx [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+  for (const namespace in layers) {
+    historyList.push({ namespace: namespace, path: path });
+    path += '/' + namespace;
+  }
+
+  const breadcrumbs: JSX.Element[] = [
+    <span className={classes(actionBarCss.workspace)} key='workspace'>
+      {'Layers'}
+    </span>,
+    <aside key='workspace-sep' className={actionBarCss.workspaceSep} />,
```
[the remark this answers is not in the document]
Comment (Bobgy) [reply frontend/src/components/graph/SubDagLayer.tsx:121]: > Asides are frequently presented as sidebars or call-out boxes.

More info here: https://www.w3.org/WAI/GL/wiki/Using_HTML5_aside_element


Therefore, I think there is no need to be fancy, if we do not have a good semantic html tag for it. We can just use the plain old `div`.

[the remark this answers is not in the document]
Comment (zijianjoy) [reply frontend/src/components/graph/SubDagLayer.tsx]: Sounds good, updated to use the plain old `div`.

**Comment by zijianjoy** [comment]
Merging it for the completion of comments. Feel free to continue with the discussion! Thank you for the input!

**Pull request merged**